arithmetic_analysis/jacobi_iteration_method.py

```diff
@@ -130,8 +131,45 @@ def jacobi_iteration_method(
             temp = (temp + val) / denom
             new_val.append(temp)
         init_val = new_val
+    """
+
+    """
+    denom - a list of values along the diagonal
+    """
+    denom = np.diag(coefficient_matrix)
+    """
+    val_last - values of the last column of the table array
+    """
+    val_last = table[:, -1]
+    """
+    masks - boolean mask of all strings without diagonal
+    elements array coefficient_matrix
+    """
+    masks = ~np.eye(coefficient_matrix.shape[0], dtype=bool)
+    """
+    no_diag - coefficient_matrix array values without diagonal elements
+    """
+    no_diag = coefficient_matrix[masks].reshape(-1, rows - 1)
+    """
+    Here we get 'i_col' - these are the column numbers, for each row
+    without diagonal elements, except for the last column.
+    """

```

Comment:
Please use real variable names: `denom` -> `denominator`, `no_diag` --> `no_diagonals` (some readers could assume `no_diagnostics`).  Is there a better variable name than `temp`?  Using real names will make the code more self-documenting and will mean that fewer comments will be required to make things clear to your reader.

Reply:
@cclauss  changed variable names. But temp is a multifaceted task, I made this name not too long, if everything is described in its name it will turn out to be a completely cumbersome name. Replaced comments with single-row ones.

Why did I use multi-line comments in the previous version: since not everything fit, and I didn’t want to comment several lines with single-line comments.

Reply:
```python
# There is nothing wrong with really long comments if they are appropriately wrapped at
# 88 characters per line.
```